ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/policyGenerator.go

```diff
@@ -15,19 +15,33 @@ var sourcePath string
 var tempPath string
 var outPath string
 var stdout bool
+var isTestRun bool
```

Comment:
I believe we shouldn't let the test flag come from end user. The test mode should be independent

Reply:
I agree I dont want to add this flag the only issues is the `os.Executable()` call in fileHandler.go that I need to change to a os.getwd() for the test runs. The rest of the changes are just a way to enable that if you have a different suggestion I would be happy to change this approach. 

I considered mocking the fileHandler.go completely for the tests but the policyGenerator doesnt reference a fileHandler as a class variable rather it is encapsulated inside policyBuilder.

Reply:
may consider failing safe, if reading from os.Excutable() fail then read from os.getWd()

Reply:
ohh that sounds reasonable and will work with the actual run as well! I can make that change! thanks for the suggestion! :)

Reply:
updated